Makefile

```diff
@@ -0,0 +1,13 @@
+JAR ?= ./target/clines-api-0.0.1-SNAPSHOT.jar
+
+package:
+	$(info Packaging application)
+	@ ./mvnw clean package
+
+docker/build: package $(JAR)
+	$(info Building docker image)
+	@ docker image build --build-arg JAR=$(JAR) -t caelum/clines-api:latest .
+
+run: docker/build
+	$(info Starting containers)
+	@ docker-compose up -d
```

Comment:
💅  It's a good thing to leave a \n in the end of the file, so future changes won't make git think this line changed when this really didn't change